### DISCUSSION TITLE
How to unblock crosh or CHrome://flags

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1205] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
How to unblock crosh or CHrome://flags  or install blocked extensiosn
Info:
Device:Hp Fortis 11 g10

crosh blocked

javascript blocked

chrome version 125 

has access to rig tools.

**Comment by schoolexploitkid** [comment]
is it that is you type crosh it auto-close or it says "blocked by admin"

**Comment by ghost** [comment]
when I hit Ctrl alt t to open it says crosh Is blocked by admin and when I open the link in chrome it says "Application blocked by your administrator
This application has been blocked by your administrator


what do I do

**Reply by schoolexploitkid** [reply, marked as answer]
u cant bypass that bc block by policy

**Reply by ghost** [reply]
ok thanks, I'm a little dumb
